web/client/translations/data.zh-ZH.json

```diff
@@ -1107,6 +1107,7 @@
             "showPreview": "Show preview",
             "advancedSettings": "Advanced Settings",
             "templateMetadataAvailable": "Metadata available from Dublin Core format: abstract, boundingBox, contributor, creator, description, format, identifier, references, rights, source, subject, temporal, title, type, uri",
+            "invalidUrlHttpProtocol": "The URL doesn't match the MapStore HTTP protocol {protocol}",
```

Comment:
We usually translate only the default languages supported by mapstore DE, US, ES, FR and IT so this could be removed